hallo_statlib/conanfile.py

```diff
@@ -41,15 +41,17 @@ def build(self):
         #
         # ***************************************************************************************
 
-        WORK_SPACE = "../workspace"
-        PROJECT_DIR = "../workspace/hello_statlib"
+        WORK_SPACE = "workspace"
+        PROJECT_DIR = "workspace/hello_statlib"
         PROJECT_NAME = "hello_statlib"
-        BUILD_TYPE = "Debug"
+        BUILD_TYPE = str(self.settings.build_type)
         
-        cmd = "eclipse -nosplash -application org.eclipse.cdt.managedbuilder.core.headlessbuild -data " + WORK_SPACE + " -import " + PROJECT_DIR + " " + PROJECT_NAME + "/" + BUILD_TYPE
-        self.run("cmd")
+        cmd = ("eclipse -nosplash -application org.eclipse.cdt.managedbuilder.core.headlessbuild -data "
+               + WORK_SPACE + " -import " + PROJECT_DIR + " -debug")
+        self.run(cmd)
 
     def package(self):
+        # I would recommend "include", "lib"
```

Comment:
For dest. Default ``package_info()`` adds "include" to include paths, "lib" to lib paths. If you are using  a different layout, just remember to set them too, something like:
```python
def package_info(self):
     self.cpp_info.includedirs= ["inc"]
     self.cpp_info.libdirs = ["bin"]
     ...
```